README.md

```diff
@@ -192,6 +192,22 @@ Command Options
 | `--solution=$solution`            | Set Visual Studio Solution version (win)
 
 
+Configuration
+--------
+
+#### `node-gyp` responds to environment variables:
+Variables take the form `npm_config_OPTION_NAME` for any of the options
+listed above.<br>
+They can be set in two ways:
+1. as `npm` configuration - these work only when `node-gyp` is triggered by
+ `npm`. for example:<br>
+ `$ npm config set [--global] npm_config_devdir /tmp/.gyp`
+2. as environment variables - these work also when `node-gyp` is invoked
+ directly:<br>
+ `> set npm_config_devdir=c:\temp\.gyp` for Windows or<br>
+ `$ export npm_config_devdir=/tmp/.gyp` for unix like environments
```

Comment:
> is the #### at the top necessary?
 In the other suctions the sub-title is followed by a table. I thought here it should be delineated in some way 

> maybe s/triggered/executed/? 'triggered' is somewhat unspecific.

`npm` -> `node-gyp` is sort of a trigger... but I changed it anyway.